mozilla-mobile/fenix/app/src/main/res/values/strings.xml

```diff
@@ -620,6 +620,8 @@
     <string name="onboarding_customize_theme_main_image_content_description">Large header image of the theme selection view showing illustrated artist and builder tools.</string>
     <!-- Onboarding theme selection card content description for the theme selection image. %1$s is placeholder for either 'system', 'light' or 'dark'. -->
     <string name="onboarding_customize_theme_content_description">Image of the %1$s theme</string>
+    <!-- Action label for options that can be selected. Talkback will append this to say "Double tap to select". -->
+    <string name="onboarding_customize_theme_a11y_action_label_select">select</string>
```

Comment:
@RebecaTudor I saw there was a discussion with Flod about this string [here](https://phabricator.services.mozilla.com/D241988), but I was on PTO and don’t see a clear resolution to his last comment. Was there perhaps a side conversation where it was agreed to land the string as is?

Reply:
There is also the conversation in the diff, that might not be very visible.
https://phabricator.services.mozilla.com/D241988#8354085

> Not blocking as there are at least 5 other strings following this structure

I don't think you can avoid using the OS strings for the first part without hacks, and the alternatives have other issues.

Reply:
Missed that comment, thanks!